pkg/kapp/config/default.go

```diff
@@ -130,6 +130,12 @@ rebaseRules:
   resourceMatchers:
   - apiVersionKindMatcher: {apiVersion: v1, kind: Pod}
 
+- path: [metadata, annotations, "kapp.k14s.io/last-renewed-time"]
+  type: copy
+  sources: [new, existing]
```

Comment:
Do we see this annotation being set by the user? If yes, in what scenarios. If no, then we can always do `sources: [existing]`